pkg/controller/installation/core_controller.go

```diff
@@ -194,6 +194,12 @@ func add(mgr manager.Manager, r *ReconcileInstallation) error {
 		return fmt.Errorf("tigera-installation-controller failed to watch primary resource: %w", err)
 	}
 
+	// Watch for changes to TigeraStatus.
+	err = c.Watch(&source.Kind{Type: &operator.TigeraStatus{ObjectMeta: metav1.ObjectMeta{Name: "calico"}}}, &handler.EnqueueRequestForObject{})
```

Comment:
Please extract a constant for "calico" and use it throughout this file.

Reply:
Done